src/main.rs

```diff
@@ -498,6 +507,63 @@ fn run_run(cx: &Context) -> Result<()> {
     Ok(())
 }
 
+fn run_afl(cx: &Context) -> Result<()> {
+    let mut cargo = cx.cargo();
+
+    set_env(cx, &mut cargo, IsNextest(false))?;
+
+    let input_dir = "inputs/";
+    let output_dir = "outputs/";
+
+    fs::create_dir_all(input_dir)?;
+    fs::create_dir_all(output_dir)?;
+
+    if !cx.args.ignore_run_fail {
+        {
+            let mut cargo = cargo.clone();
+
+            cargo.arg("afl").arg("build");
+            cargo.arg("--target-dir").arg(cx.ws.target_dir.as_str());
+            if term::verbose() {
+                status!("Running", "{cargo}");
+                cargo.stdout_to_stderr().run()?;
+            } else {
+                // Capture output to prevent duplicate warnings from appearing in two runs.
+                cargo.run_with_output()?;
+            }
+        }
+
+        // prepare seeds
+        let seeds = vec!["example input 1", "sample data 2", "test case 3"];
+
+        for (index, seed) in seeds.into_iter().enumerate() {
+            let file_path = format!("{input_dir}seed_{index}.txt");
+            let path = Path::new(&file_path);
+            let mut file = fs::File::create(path)?;
+            file.write_all(seed.as_bytes())?;
+        }
```

Comment:
I beliave these should not be defined on our end, but should respect what the user passes as arguments.

Reply:
yeah, I would fix it,